tempus_fugit_minecraft/model.py

```diff
@@ -81,6 +81,7 @@ def __init__(self) -> None:
         self.queue = deque()
 
         self.player = Player()
+        self.trees = list()
```

Comment:
Do we actually need to store the trees as a separate list? No other structures (clouds mainly) are currently stored outside of the `world`. Plus if the player breaks a tree block, the list becomes useless anyway since the list is no longer accurate to the current `world`.

Reply:
Good point.. It doesn't make sense to make a list of trees while they are stored in `model.world`.
However, I made `generate_trees()` to return a list of trees. That was mainly for linking each tree with its corresponding items' coordinates, PLUS for testing purposes.

I will update them.
Thanks.